### ISSUE TITLE
External Text Content

### ISSUE THREAD

**Issue by biggs3d** [Neurobi/Neurobi.github.io#14]
HTML:
`<div id="tempDIv" style="display:none;"></div>`
`<p id="text"></p>`

JQuery:
` $('#tempDiv').load("pathtofile/content.txt");`
` $('#text').html($('#tempDiv').find('#theParagraphId').html());`

[https://stackoverflow.com/questions/20544219/loading-external-text-from-txt-to-a-html-file/20544308](https://stackoverflow.com/questions/20544219/loading-external-text-from-txt-to-a-html-file/20544308)

**Comment by asa55** [comment]
I think I like where this is going - if I get what you're saying correctly, we can replace many of the static content .js files with .txt files. This would let us format the HTML in each .txt file nicely, instead of the current implementation where all the .js files have HTML in one single (long) character string. Is that about right or were you headed somewhere different with this?

**Comment by biggs3d** [comment]
Yeah, it'd be so much easier to just embed, but if you really want it separate than this might be a whole lot easier to not only create, but search/read/modify. I'd definitely embed empty space though!

A concern however is this uses jquery which is a lot of overhead, would have to test/research :-/

Your naming convention for tables is very precise and snaps together well, but to make it human readable using (or tacking on) something like "bio_aa" would help navigate the doc too. Currently, shifting say... my bio blurb to the right a column would not be totally straightforward, or say in the case you moved some text outside a table structure for some reason. Or... hhmmm, guess I'm trying to say let the naming convention reflect the type of content, since that will not change, versus it's place in the world, which may.
